api.raml

```diff
@@ -33,6 +33,12 @@ baseUri: http://api.circular.works
                             "id": "25"
                           }
                         ]
+                      },
+                      "author": {
+                        "data": {
+                          "type": "users",
+                          "id": "3"
+                        }
```

Comment:
Do we need the author populated on the index endpoint? Looks like the kind of information that should be loaded when the user is visiting the `show#task` endpoint